bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
-        items:
-          $ref: '#/definitions/CustomerFieldReferenceDto'
-      requirements:
+        description: 'Specifies custom message on ticket when customer does not want pick-up. May contain HTML markup.'
+      ticketMsg:
         type: string
-        description: 'A text describing what you need to bring for this activity. May contain HTML.'
-      route:
-        $ref: '#/definitions/ActivityRouteDto'
-        description: 'Optional travel directions that show the start point, end point, and any waypoints in between.'
-      seasonAllYear:
-        type: boolean
-        description: 'Deprecated.'
-      seasonEndDate:
-        type: integer
-        format: int32
-        description: 'Deprecated.'
-      seasonEndMonth:
-        type: integer
-        format: int32
-        description: 'Deprecated.'
-      seasonStartDate:
-        type: integer
-        format: int32
-        description: 'Deprecated.'
-      seasonStartMonth:
-        type: integer
-        format: int32
-        description: 'Deprecated.'
-      selectFromDayOptions:
-        type: boolean
-        description: 'Deprecated.'
+        description: 'Specifies custom message on ticket. May contain HTML markup.'
       showGlobalPickupMsg:
         type: boolean
+        description: 'Specifies if the custom message is turned on to be shown on ticket.'
       showNoPickupMsg:
         type: boolean
-      slug:
-        type: string
-        description: 'Human readable string that may be used to identify the product.'
-      startTimes:
+        description: 'Specifies if the custom message when customer does not want pick-up is turned on to be shown on ticket.'
+      pickupPlaceGroups:
+        type: object
+        description: 'Not working. The list of the pickup place groups for this activity.'
+      dropoffService:
+        type: boolean
+        description: 'Specifies whether this activity offers drop off service.'
+      dropoffFlags:
         type: array
-        description: 'Only used if bookingType == DATE_AND_TIME. A list of scheduled start times for this tour/activity.'
         items:
-          $ref: '#/definitions/StartTimeDto'
-      tagGroups:
+          type: string
+        description: 'The list of flags added to the drop-off of this activity.'
+      customDropoffAllowed:
+        type: boolean
+        description: 'Only relevant if dropoffService is TRUE. Specifies whether the customer should be allowed to enter their drop off place in free text.'
+      useSameAsPickUpPlaces:
+        type: boolean
+        description: 'Not working. Specifies if supplier wants to drop-off users to the same places that they can be picked up from.'
+      dropoffPlaceGroups:
+        type: object
+        description: 'Not working. The list of the drop-off place groups for this activity.'
+      difficultyLevel:
+        type: string
+        description: 'Specifies how physically demanding this tour/activity is.'
```

Comment:
This is an enum of 
VERY_EASY("0"), EASY("1"), MODERATE("5"), CHALLENGING("2"), DEMANDING("3"), EXTREME("4");
(double check if we are using strings or numbers)

Reply:
It is string type. Fixed.